Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Controllers/MixedRealityControllerInfo.cs

```diff
@@ -4,13 +4,13 @@
 using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.SDK.UX.MotionController
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Controllers
 {
     /// <summary>
     /// This script keeps track of the GameObjects representations for each button on the Windows Mixed Reality Controllers.
     /// It also keeps track of the animation Transforms in order to properly animate according to user input.
     /// </summary>
-    public class WindowsMixedRealityControllerInfo
+    public class MixedRealityControllerInfo
```

Comment:
Isn't this specific to Windows Mixed Reality?

Reply:
Either the class name should be changed or the summary updated.

Reply:
No longer, this visualizer works for all platforms now!
But your right, that summary tag should be updated